lib/logstash/runner.rb

```diff
@@ -63,15 +44,6 @@ def run(args)
         end
         return LogStash::Agent.run($0, agent_args)
       end,
-      "rspec" => lambda do
```

Comment:
ah yes, I kind of unilaterally decided that this did not really belong into the runner. we can put it back if that's a problem.